robottelo/constants/__init__.py

```diff
@@ -283,9 +283,10 @@ class Colored(Box):
     'rhel7': 'Red Hat Enterprise Linux 7 Server (RPMs)',
     'rhva6': ('Red Hat Enterprise Virtualization Agents for RHEL 6 Server (RPMs)'),
     'rhs7': 'Red Hat Satellite 6.11 (for RHEL 7 Server) (RPMs)',
-    'rhsc7': 'Red Hat Satellite Capsule 6.9 (for RHEL 7 Server) (RPMs)',
+    'rhs8': 'Red Hat Satellite 6.13 for RHEL 8 x86_64 (RPMs)',
+    'rhsc7': 'Red Hat Satellite Capsule 6.11 (for RHEL 7 Server) (RPMs)',
+    'rhsc8': 'Red Hat Satellite Capsule 6.13 for RHEL 8 x86_64 (RPMs)',
     'rhsc7_iso': 'Red Hat Satellite Capsule 6.4 (for RHEL 7 Server) (ISOs)',
```

Comment:
We can not remove the 6.4 ISO since the `tests/foreman/api/test_capsulecontent.py::test_positive_iso_library_sync` uses it unless @vsedmik and his team decide to use a different ISO repository.
I would keep this one here since we do not install the ISO contents, but use it for content testing.

Reply:
Ack, thank you both for clarifying that.